platform-ui-skin/src/main/webapp/skin/less/variables.less

```diff
@@ -60,6 +60,7 @@
 @sizeIcon64x64: 60px;
 @sizeIcon32x32: 30px;
 @sizeIconDefault: 18px;
+@textHeaderFontSize: 18px;
```

Comment:
please place it near other text font sizes, by example
https://github.com/Meeds-io/platform-ui/blob/7ea73c9444eea50b6451cf98275df4b6f007872e/platform-ui-skin/src/main/webapp/skin/less/variables.less#L252-L253

Reply:
done!